webpack.config.js

```diff
@@ -37,7 +37,7 @@ const baseConfig = (isProd, minify) => ({
             {
                 test: /\.less$/,
                 use: [
-                    MiniCssExtractPlugin.loader,
+                    isProd ? MiniCssExtractPlugin.loader : "",
```

Comment:
I noticed the commit message for this was to "fix webpack.config.js for windows". I'm curious what was happening initially (for the record, this looks good. I'm just personally curious).

Reply:
When loading the design site locally none of the css would load with the following error:
```
Refused to apply style from 'http://localhost:8080/xxxxxx' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
```
Giamir gave me the solution to disable the css plugin locally as he encountered the same issue when assisting Tavian.

I believe it may be related to this as the root cause: https://github.com/webpack/webpack-dev-server/issues/3168#issuecomment-816580916